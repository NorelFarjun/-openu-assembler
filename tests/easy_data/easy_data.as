.data -100, 100, 0, 1000,-1000
