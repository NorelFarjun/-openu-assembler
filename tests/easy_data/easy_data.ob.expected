0 5
0100 77634
0101 00144
0102 00000
0103 01750
0104 76030
